packages/editor/src/components/media-upload/check.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * WordPress dependencies
+ */
+import { withSelect } from '@wordpress/data';
+
+export function MediaUploadCheck( { hasUploadPermissions, fallback, children } ) {
```

Comment:
This line can be replaced by adding a default value to the component definition:

```diff
- export function MediaUploadCheck( { hasUploadPermissions, fallback, children } ) {
- 	const optionalFallback = fallback || null;
- 	return hasUploadPermissions ? children : optionalFallback;
+ export function MediaUploadCheck( { hasUploadPermissions, fallback = null, children } ) {
+	return hasUploadPermissions ? children : fallback;
```